contents/articles/2016-10_yamen_hmd_calibration/index.md

```diff
@@ -0,0 +1,22 @@
+--- 
+title: Layered Telepresence
+date: Thu Oct 20 2016 14:14:11 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Yamen Saraiji have developed a Layered Telepresence system which is a layered-based approach to create a seamless interaction between multiple users from multiple locations to engage in the same conversation...."
+featured_img_thumb: ""
+---
+
+[Yamen Saraiji](http://myamens.com/) have developed a Layered Telepresence system which is a layered-based approach to create a seamless interaction between multiple users from multiple locations to engage in the same conversation.
+
+The system combines multiple locations and arrange them as layers that are presented in the same virtual space. The concept allows users to achieve multiple types of compound presence in both physical and virtual environments.
+
```

Comment:
@nathakits  - need to talk about how eye tracking is relevant. For example: 

Yamen and his team use eye tracking to determine the user's gaze position; which in turn is used to inform the saliency of the layer. Pupil hardware is used in the VR headsets. Yamen developed a Unity3d plugin, that enables talks to Pupil Service over the network and coordinates calibration within the VR headset. Check out Yamen's Unity3d calibration plugin here on [github].